### PR TITLE
test: include jest-axe to test that components do not return a11y violations

### PR DESCRIPTION
**Pull request by scurker**
This should hopefully provide some better coverage of finding basic a11y violations if they're accidentally introduced.

### CODE REVIEW

File: __tests__/src/components/Alert/index.js [hunk 1]
```diff
@@ -67,3 +68,8 @@ test('supports the "contentRef" prop', () => {
   expect(called).toBeTruthy();
   alert.unmount();
 });
+
+test('should return no axe violations', async () => {
```
Comment (jkodu) [inline __tests__/src/components/Alert/index.js]: for each of the component, can we add tests, where there are violations, just so we know this works either way.

Comment (schne324) [reply __tests__/src/components/Alert/index.js]: don't we know that axe works? It seems like you're asking for a test that confirms that axe works

**Comment by schne324** [comment]
looks like the test might be failing due to some case-sensitivity (`Sidebar` vs `SideBar`)

File: __tests__/src/components/Alert/index.js (hunk 1 above)
Comment (scurker) [reply __tests__/src/components/Alert/index.js]: I did confirm that violations will result in a failure. Any particular reason we would want to verify that it does return violations?

Comment (scurker) [reply __tests__/src/components/Alert/index.js]: `jest-axe` already has tests in place https://github.com/nickcolley/jest-axe/blob/37384ae90c93723c85785bc0bdb96a4b3e512766/index.test.js#L67-L72

**Pull request merged**